docs/blog/_posts/2018-07-06-ninth-dotty-milestone-release.md

```diff
@@ -0,0 +1,163 @@
+support for sbt 1, introduced improved unchecked warnings and improved SAM type support.
+
+## What’s new in the 0.9.0-RC1 technology preview?
+
+### Improved REPL [#4680](https://github.com/lampepfl/dotty/pull/4680)
+The REPL now uses [JLine 3](https://github.com/jline/jline3) under the hood which improves on
+many aspects such as auto-completions and multi-line editing. The REPL now also works on Windows!
+
+### Documentation support in the IDE [#4461](https://github.com/lampepfl/dotty/pull/4461), [#4648](https://github.com/lampepfl/dotty/pull/4648)
+The Dotty IDE will now display documentation on code hover for symbols that were previously
+compiled by the Dotty compiler. In the future, we plan to let users query the documentation
+in the REPL as well.
+
+### Drop requirement that implicit functions must be non-empty [#4549](https://github.com/lampepfl/dotty/pull/4549)
+We decided to remove an arbitrary restriction that implicit and functions must be non-empty.
+We can now write:
+```scala
+type IntProducer = implicit () => Int
+
+def prod1: IntProducer = 1
+val prod2: IntProducer = 2
+```
+
+An interesting observation is that by-name parameters can now be encoded as implicit function types:
+```scala
+def timed[T](op: => T) = ...
+def timed[T](op: implicit () => T) = ...
+
+timed {
+  fetch(url)
+}
+```
+
+Both definitions above are equivalent.
+
+### Emit feature warnings for implicit conversions [#4229](https://github.com/lampepfl/dotty/pull/4229)
+Implicit conversions are easily the most misused feature in Scala. We now emit feature warnings
+when encountering an implicit conversion definition, just like Scala 2 does.
+
+In addition, we also emit a feature warning when an implicit conversion is used,
+unless the conversion is an implicit class, or otherwise co-defined with the type
+to which it converts, or the conversion is predefined in `scala.Predef` or is the
+`scala.reflect.Selectable.reflectiveSelect` conversion (we might extend this to more conversions).
+
+### Optimise s and raw interpolators [#3961](https://github.com/lampepfl/dotty/pull/4229)
+`s` and `raw` string interpolators were known to be slower than their not type-safe counterparts:
+```scala
+s"Hello $name!"
+
+// versus
+"Hello " + name + "!"
+```
+The compiler will now desugar the former into the later. Special thanks to
+[Wojtek Swiderski](https://github.com/Wojtechnology) who contributed this feature to the Dotty
+compiler!
+
+## Trying out Dotty
+
+### sbt
+Using sbt 1.1.4 or newer, do:
+
+```shell
+sbt new lampepfl/dotty.g8
+```
+
+This will setup a new sbt project with Dotty as compiler. For more details on
+using Dotty with sbt, see the
+[example project](https://github.com/lampepfl/dotty-example-project).
+
+### IDE support
+It is very easy to start using the Dotty IDE in any Dotty project by following
+the [IDE guide](https://dotty.epfl.ch/docs/usage/ide-support.html).
```

Comment:
Nitpick: I remember from a very old tutorial on writing that it is a very good practice to avoid `very` in writing because it is not very useful.

Reply:
"not very useful"? 😄 😆

Reply:
Well maybe it was easy before, but now it's even easier, i.e. *very* easy :D

Reply:
Very good comment.

Reply:
"simply", "obviously", "very", "easily" are generally best avoided. They serve little purpose but to make people feel like idiots when they don't understand or fail to follow the instructions.